DelegateFun/Sorter/SortUtility.cs

```diff
@@ -4,7 +4,32 @@ namespace Sorter
 {
     public class SortUtility
     {
-        // Sort method should be implemented here
-        // It should accept an int[] and a delegate you define that performs the actual comparison
+        public delegate bool Analyze(int x, int y);

```

Comment:
I might suggest renaming the delegate to something like "AnalyzingFunction" to be a little more intent revealing, but not really an issue
```suggestion
        public delegate bool AnalyzingFunction(int x, int y);
```